spec/spec_helper.rb

```diff
@@ -13,6 +13,11 @@ def x2t
                    tmp_path: StaticData::TMP_DIR)
 end
 
+def xml
```

Comment:
I think those lines should not be in `spec_helper` but you can place it in `framework/x2t.rb`